[2025-05-07T19:31:46.055+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Simple_Corona_project.Spark_Processing manual__2025-05-07T19:31:44.010759+00:00 [queued]>
[2025-05-07T19:31:46.062+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Simple_Corona_project.Spark_Processing manual__2025-05-07T19:31:44.010759+00:00 [queued]>
[2025-05-07T19:31:46.062+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-05-07T19:31:46.074+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): Spark_Processing> on 2025-05-07 19:31:44.010759+00:00
[2025-05-07T19:31:46.078+0000] {standard_task_runner.py:60} INFO - Started process 223 to run task
[2025-05-07T19:31:46.081+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Simple_Corona_project', 'Spark_Processing', 'manual__2025-05-07T19:31:44.010759+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/Project_tasks2.py', '--cfg-path', '/tmp/tmpx8u2h_vl']
[2025-05-07T19:31:46.082+0000] {standard_task_runner.py:88} INFO - Job 33: Subtask Spark_Processing
[2025-05-07T19:31:46.126+0000] {task_command.py:423} INFO - Running <TaskInstance: Simple_Corona_project.Spark_Processing manual__2025-05-07T19:31:44.010759+00:00 [running]> on host 1a616c0a8036
[2025-05-07T19:31:46.195+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Simple_Corona_project' AIRFLOW_CTX_TASK_ID='Spark_Processing' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T19:31:44.010759+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-07T19:31:44.010759+00:00'
[2025-05-07T19:31:46.196+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-07T19:31:46.197+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        cd /opt/***/dags/CoronaProj/\n        spark-submit --jars "postgresql-42.7.5.jar" Spark_script_process.py']
[2025-05-07T19:31:46.205+0000] {subprocess.py:86} INFO - Output:
[2025-05-07T19:31:48.548+0000] {subprocess.py:93} INFO - 25/05/07 19:31:48 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-07T19:31:48.890+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-05-07T19:31:48.891+0000] {subprocess.py:93} INFO -   File "/opt/***/dags/CoronaProj/Spark_script_process.py", line 2, in <module>
[2025-05-07T19:31:48.892+0000] {subprocess.py:93} INFO -     import findspark
[2025-05-07T19:31:48.892+0000] {subprocess.py:93} INFO - ModuleNotFoundError: No module named 'findspark'
[2025-05-07T19:31:48.901+0000] {subprocess.py:93} INFO - 25/05/07 19:31:48 INFO ShutdownHookManager: Shutdown hook called
[2025-05-07T19:31:48.902+0000] {subprocess.py:93} INFO - 25/05/07 19:31:48 INFO ShutdownHookManager: Deleting directory /tmp/spark-3194cabe-a429-4f39-8a9f-337ad6523557
[2025-05-07T19:31:48.938+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-05-07T19:31:48.948+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-05-07T19:31:48.950+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=Simple_Corona_project, task_id=Spark_Processing, execution_date=20250507T193144, start_date=20250507T193146, end_date=20250507T193148
[2025-05-07T19:31:48.964+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 33 for task Spark_Processing (Bash command failed. The command returned a non-zero exit code 1.; 223)
[2025-05-07T19:31:48.983+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-05-07T19:31:49.006+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
